packages/next/src/server/future/helpers/module-loader/node-module-loader.ts

```diff
@@ -7,7 +7,10 @@ export class NodeModuleLoader implements ModuleLoader {
   public async load<M>(id: string): Promise<M> {
     if (process.env.NEXT_RUNTIME !== 'edge') {
       // Need to `await` to cover the case that route is marked ESM modules by ESM escalation.
-      return await require(id)
+      return (await process.env.NEXT_MINIMAL)
+        ? // @ts-ignore
+          __non_webpack_require__(id)
+        : require(id)
```

Comment:
Should be this (or just remove `await` as this is already async)?

```suggestion
      return await (process.env.NEXT_MINIMAL
        ? // @ts-ignore
          __non_webpack_require__(id)
        : require(id))
```